Winapp2.ini

```diff
@@ -15190,17 +15359,20 @@ Default=False
 FileKey1=%CommonAppData%\Sendori|*.log
 
 [Serious Sam *]
-Section=Games
-Detect1=HKCU\Software\Valve\Steam\Apps\41014
-Detect2=HKCU\Software\Valve\Steam\Apps\41070
-Detect3=HKCU\Software\Valve\Steam\Apps\564310
+Detect1=HKCU\Software\Valve\Steam\Apps\41000
+Detect2=HKCU\Software\Valve\Steam\Apps\41010
+Detect3=HKCU\Software\Valve\Steam\Apps\41014
+Detect4=HKCU\Software\Valve\Steam\Apps\41050
+Detect5=HKCU\Software\Valve\Steam\Apps\41060
+Detect6=HKCU\Software\Valve\Steam\Apps\41070
+Detect7=HKCU\Software\Valve\Steam\Apps\204340
+Detect8=HKCU\Software\Valve\Steam\Apps\227780
+Detect9=HKCU\Software\Valve\Steam\Apps\564310
 DetectFile=%ProgramFiles%\Croteam\Serious Sam *

```

Comment:
DetectFile should also be removed if the old FileKeys 1+2 are removed.

Reply:
Do you know what those original FileKeys were? Were they for CD copies or something? None of the Serious Sam games I have had anything in those directories.

Reply:
Unfortunately no, but I think you're right that it was for the CD/DVD version from the pre-Steam period.